tgui/packages/tgui/interfaces/CrewCredits.tsx

```diff
@@ -0,0 +1,82 @@
+import { BooleanLike } from 'common/react';
```

Comment:
Can we get a file license header in here? I know some are missing them, but I'm trying to get all of our tgui interfaces to have licenses headers. See a bunch of the other interfaces for examples. MIT or ISC are common and work fine for us, but you can pick any license that will allow us to use it.